client/src/components/TaskDurationBarChart/TaskDurationBarChart.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import styles from "./TaskDurationBarChart.module.css";
+import PropTypes from "prop-types";
+import {
+  BarChart,
+  Bar,
+  XAxis,
+  YAxis,
+  ReferenceLine,
+  ResponsiveContainer,
+} from "recharts";
+
+export const TaskDurationBarChart = ({ taskResponseData }) => {
+  const taskDurationDifferences = [];
+
+  console.log("hi");
+  console.log(taskResponseData);
+  for (const task of taskResponseData) {
+    const taskDifference = {
+      taskName: task.title,
+      percentDifference: 0,
+    };
+
+    console.log(task);
+    const plannedDuration =
+      task.originalEndDate.getTime() - task.originalStartDate.getTime();
+    const actualDuration = task.endDate.getTime() - task.startDate.getTime();
+    taskDifference.percentDifference =
+      (actualDuration - plannedDuration) / plannedDuration;
+
+    if (task.taskStatus === "DONE")
```

Comment:
Minor comment: This is fine for now, although later on we'll need to change this to `task.isStarted/endedDate` once we start linking it to the backend

Reply:
Noted